src/skeleton/skeleton.ts

```diff
@@ -0,0 +1,22 @@
+import { html, LitElement, nothing } from 'lit';
+import { property } from 'lit/decorators.js';
+import { SkeletonAnimation, SkeletonVariant } from './types';
+
+export class Skeleton extends LitElement {
+  @property({ reflect: true }) variant?: SkeletonVariant = 'line';
+  @property({ reflect: true, attribute: 'animation-mode' })
+  animationMode?: SkeletonAnimation = 'progress';
```

Comment:
The animations are not defined in design system but i think we can merge this and change the animation according the design system in future.